golang/cosmos/x/vtransfer/ibc_middleware_test.go

```diff
@@ -0,0 +1,23 @@
+package vtransfer_test
+
+import (
+	"encoding/json"
+
+	ibctesting "github.com/cosmos/ibc-go/testing"
```

Comment:
Oh, okay.  I assume `ibc-go/v4/testing` doesn't exist?

Reply:
It does wrong import need it for tests